transform/surveys/144.0001.json

```diff
@@ -0,0 +1,52 @@
+{
+  "title": "UKIS",
+  "survey_id": "144",
+  "form_type": "0001",
+  "question_groups": [
+    {
+      "title": "3. Innovation investment",
+      "questions": [
+        {
+          "text": "3.3 Please estimate the amount of expenditure on internal Research & Development  the calendar  year 2016 only.",
```

Comment:
Extra spaces between 'Development the calendar year in this and the following line. Also the word 'in' is in one not the other.